src/pom.xml

```diff
@@ -1791,7 +1791,7 @@
   <jetty.version>9.2.13.v20150730</jetty.version>
   <jetty.servlet-api.version>3.1.0</jetty.servlet-api.version>
   <poi.version>3.14</poi.version>
-  <wicket.version>7.1.0</wicket.version>
+  <wicket.version>7.2.0</wicket.version>
```

Comment:
We should probably discuss changing wicket versions at this stage in the release process.